docs/development/hosting.md

```diff
@@ -46,11 +27,9 @@ The full publishing flow is as follows:
    does any internal checks.  These checks may include branch checks,
    verification that an authorized user made the commit, checks for passing
    tests, or other checks.
-0. Jenkins will set the `TESTPILOT_AMO_USER` and `TESTPILOT_AMO_SECRET`
-   environment variables
-0. Jenkins will run `bin/build-addon.sh`
-0. Jenkins will prepare appropriate S3 files (the signed .xpi file, /latest/
-   redirect, a new updates.json file)
+0. Jenkins will run `npm install` followed by `npm run package`
```

Comment:
FWIW, I'd be a *little* hesitant to require an npm script versus a plain shell script. Mozilla Extensions (nee bootstrapped extensions) don't really need node & npm & package.json to produce an .xpi - just zip. 

Maybe practically speaking this is fine, though.  While rewriting the Test Pilot add-on for #2423, I ended up backing into a webpack build anyway - mainly because JavaScript Module imports are clunky IMO